### PR TITLE
Readd @markolazic88

### PR DESCRIPTION
**Pull request by markolazic88**
Following up #505

### CODE REVIEW

File: src/Firehose.Web/Authors/MarkoLazic.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        public Uri WebSite => new Uri("https://markolazic.com/");
+
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("https://markolazic.com/feed/"); }
+        }
+
+        public string GitHubHandle => "markolazic88";
+
+        public bool Filter(SyndicationItem item)
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/MarkoLazic.cs]: Please remove the filter. We already filter for Xamarin in title and categories.

File: src/Firehose.Web/Authors/MarkoLazic.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class MarkoLazic : IAmACommunityMember, IFilterMyBlogPosts
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/MarkoLazic.cs]: Remove `IFilterMyBlogPosts`

File: src/Firehose.Web/Authors/MarkoLazic.cs (hunk 1 above)
Comment (markolazic88) [reply src/Firehose.Web/Authors/MarkoLazic.cs]: Removed. I just compared the class with the other IAmACommunityMember classes, and the properties match now. Thanks for the hint!

File: src/Firehose.Web/Authors/MarkoLazic.cs (hunk 2 above)
Comment (markolazic88) [reply src/Firehose.Web/Authors/MarkoLazic.cs]: Removed.

Comment (Cheesebaron) [reply src/Firehose.Web/Authors/MarkoLazic.cs]: Excellent

**Pull request merged**